ml/ad_charts.cc

```diff
@@ -11,8 +12,8 @@ void ml_update_dimensions_chart(ml_host_t *host, const ml_machine_learning_stats
             char id_buf[1024];
             char name_buf[1024];
 
-            snprintfz(id_buf, 1024, "machine_learning_status_on_%s", localhost->machine_guid);
-            snprintfz(name_buf, 1024, "machine_learning_status_on_%s", rrdhost_hostname(localhost));
+            snprintfz(id_buf, 1024, "machine_learning_status_on_%s", rrdb.localhost->machine_guid);
+            snprintfz(name_buf, 1024, "machine_learning_status_on_%s", rrdhost_hostname(rrdb.localhost));
```

Comment:
Here we are breaking pattern considering the rest of the code, please,  add `1` to variable declaration or reduce one byte here.

Reply:
We have the same situation to the rest of this file.